Signaling/app/src/main/java/com/tokbox/android/tutorials/signaling/ChatMessage.java

```diff
@@ -0,0 +1,33 @@
+package com.tokbox.android.tutorials.signaling;
+
+public class ChatMessage {
+
+    private String mMessageText;
+    private Boolean mRemote;
+
+    public ChatMessage(String messageText) {
+        mMessageText = messageText;
+        mRemote = false;
+    }
+
+    public static ChatMessage fromData(String messageData) {
```

Comment:
Instead to have this method, could be useful to have a constructor with both parameters: messageText and remote?